benchmarking-cli/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.0.0", features = ["derive"] }
 hex = { version = "0.4.0", default-features = false }
-miden-stdlib = { version = "0.5.0", package = "miden-stdlib", default-features = false }
-miden_vm = { version = "0.6.0", package = "miden-vm", features = ["concurrent"]}
+miden-air = { version = "0.7.0", package = "miden-air", default-features = false }
```

Comment:
There is a new way of adding ExecutionOptions in this version hence the need to add a dependency to miden-air now? (We didn't have it before)

Reply:
I couldn't get ExecutionOptions from the miden-vm. It looks like we don't export it